backend/src/services/user.service.ts

```diff
@@ -15,11 +16,13 @@ export class UserService {
         const secret = process.env.JWT_SECRET;
         return User.findOne({
             where: {
-                userName: loginRequestee.userName
+                [Op.or]: [
+                    {userName: loginRequestee.userName},
+                    {email: loginRequestee.email}]
```

Comment:
We could use the following, which would make the life easier of the frontend team:
```suggestion
                    {userName: loginRequestee.userName},
                    {email: loginRequestee.userNamel}]
```
If we use the userName attribute from the loginRequestee model, the frontend wouldn't not need to provide both attritbutes